org.lflang/src/org/lflang/generator/c/TypeParameterizedReactor.java

```diff
@@ -0,0 +1,67 @@
+package org.lflang.generator.c;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.lflang.ASTUtils;
+import org.lflang.InferredType;
+import org.lflang.generator.CodeBuilder;
+import org.lflang.lf.Instantiation;
+import org.lflang.lf.Reactor;
+import org.lflang.lf.Type;
+
+public record TypeParameterizedReactor(Reactor r, Map<String, Type> typeArgs) {
```

Comment:
The generated method `r()` is not descriptive. Changing this will change that, too.
```suggestion
public record TypeParameterizedReactor(Reactor reactor, Map<String, Type> typeArgs) {
```

Reply:
Change made locally (automatic refactoring).